builders/xcm/remote-evm-calls.md

```diff
@@ -0,0 +1,337 @@
+ - **transactThroughProxy**(transactAs, xcmTransaction) — similar to the **transact** extrinsic, but with **transactAs** as an additional field. This function allows the remote EVM call to be dispatched from a given account. This account needs to have set the multilocation derivative as an **any** type proxy. On the contrary, the dispatch of the remote EVM call will fail
+
+Where the inputs that need to be provided can be defined as:
+
+ - **xcmTransaction** — contains the Ethereum transaction details of the call that will be dispatched
+ - **transactAs** — account from which the remote EVM call will be dispatched. This account needs to have set the multilocation derivative as an **any** type proxy
+
+## Building a Remote EVM call through XCM {: #build-remove-evm-call-xcm}
+
+This guide covers building an XCM message for remote EVM calls using the XCM pallet from the relay chain to Moonbase Alpha. More specifically, it will use the `transact` function. The steps to use the `transactThroughProxy` function are identical. However, you'll need to provide the **transactAs** account and ensure that this account has set the multilocation derivative as an **any** type proxy.
+
+### Checking Prerequisites {: #ethereumxcm-check-prerequisites}
+
+To be able to send the extrinsic in Polkadot.js Apps from the relay chain, you need to have the following:
+
+ - An [account](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank} on the relay chain with funds (`UNIT`) to pay for the transaction fees. You can acquire some `xcUNIT` by swapping for DEV tokens (Moonbase Alpha's native token) on [Moonbeam-Swap](https://moonbeam-swap.netlify.app){target=_blank}, a demo Uniswap-V2 clone on Moonbase Alpha, and them [send them to the relay chain](/builders/xcm/xc20/xtokens/){target_blank}. On the contrary, you can [contact us](https://discord.gg/PfpUATX){target=_blank} to get some `UNIT` tokens directly
+
+ - Fund the **multilocation-derivative account**, which you can obtained by following the steps [in the next section](#calculate-multilocation-derivative){target=_blank}. The account must have enough DEV tokens (or GLMR/MOVR for Moonbeam/Moonriver)
+  to cover the cost of the XCM execution of the remote EVM call. Note that this is the account from which the remote EVM call will be dispatched. Consequently, the account must satisfy whatever conditions are required for the EVM call to be executed correctly. For example, hold any relevant ERC-20 token if you are doing an ERC-20 transfer
+ 
+!!! note
+    Suppose you are using the `transactAs` function. In that case, the **transactAs** account is the one that needs to hold the DEV tokens (or GLMR/MOVR for Moonbeam/Moonriver) to cover the cost of the XCM execution of the remote EVM call. It must satisfy whatever conditions are required for the EVM call to be executed correctly.
```

Comment:
This is incorrect, the ML-derived account pays for the fees, while the `transactAs` account needs to satisfy the conditions for the EVM call to be executed correctly

Reply:
My bad, I've fixed it in all places where this was stated and ensure that the reader understands that the `msg.sender` is this account, but the fees are paid by the ML-derivative